src/reducers/eval-frame-action-forwarder.js

```diff
@@ -1,33 +0,0 @@
-/* eslint-disable no-fallthrough */
```

Comment:
hooray, we can finally get rid of this terrible and confusing hack!! (this is main payoff of this PR)